db/primary_migrate/20230113202352_create_sp_return_logs_returned_at_date_issuer_index.rb

```diff
@@ -0,0 +1,20 @@
+class CreateSpReturnLogsReturnedAtDateIssuerIndex < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def up
+    safety_assured do
+      execute <<-SQL
+        CREATE INDEX CONCURRENTLY index_sp_return_logs_on_returned_at_date_issuer
+        ON public.sp_return_logs
+        USING btree ((returned_at::date), issuer)
+        WHERE (billable = true and returned_at IS NOT NULL);
+      SQL
+    end
+  end
+
+  def down
+    safety_assured do
```

Comment:
heads up! this is a very expensive migration, when doing the deploy I would maybe consider spinning up a single migration instance separately, and watching it to make sure the index add doesn't go poorly